apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.ts

```diff
@@ -154,16 +162,18 @@ router.post(
         `/pl/course_instance/${res.locals.course_instance.id}/instructor/instance_admin/assessments`,
       );
     } else if (req.body.__action === 'poll_lti13_assessments') {
-      serverJobOptions.description = 'Synchronize external assignment metadata from LMS';
+      serverJobOptions.description = 'Synchronize assignment and user metadata from LMS';
       const serverJob = await createServerJob(serverJobOptions);
 
       serverJob.executeInBackground(async (job) => {
         await syncLineitems(instance, job);
+        job.info('Adding students to PrairieLearn');
+        await enrollUsersFromLti13(instance);
```

Comment:
Depending on the outcome of the discussion on `enrollUsersFromLti13`, I may request that your drop this from this PR. This feels like part of the broader enrollment management revamp, and we most likely want to use an invite system here to sidestep the problem of not having a UIN at this point.

Reply:
I wrote up a plan for this in https://github.com/PrairieLearn/PrairieLearn/issues/10624. I'm going to ask you to drop this from this PR.